src/components/CivicProfileForms/HousingInfo.jsx

```diff
@@ -1,8 +1,88 @@
 // React Imports
-import React from 'react';
+import React, { useState, useEffect } from 'react';
 // MUI Imports
-import Typography from '@mui/material/Typography';
+import { TextField, Button } from '@mui/material';
+import { useCivicProfile } from '@hooks';
 
-const HousingInfo = () => <Typography>Housing Info</Typography>;
+/**
+ * HousingInfo - A form to fill out with housing security info
+ *
+ * @memberof CivicProfileForms
+ * @name HousingInfo
+ * @returns {React.JSX.Element} The HousingInfo Component
+ */
+const HousingInfo = () => {
+  const { data, add, isSuccess } = useCivicProfile();
+  const [formData, setFormData] = useState({
+    lastPermanentStreet: '',
+    lastPermanentCity: '',
+    lastPermanentState: '',
+    lastPermanentZIP: ''
```

Comment:
Are all inputs supposed to be string? Not sure if HMIS requires that or anything. If not, we can maybe change ZIP to `type="number"`.

Reply:
They are. And zip/postal code may look like a number, but it is not a number. If you can't do math with it, it's not a number. Zip codes can, and do, have '+' signs in them, which you can't put in a number.